packages/atlas/src/hooks/useFee.ts

```diff
@@ -55,8 +82,63 @@ export const useFee = <TFnName extends TxMethodName, TArgs extends Parameters<Jo
     }
     argsRef.current = args
     setLoading(true)
-    getFee(args)
-  }, [args, getFee])
+    getFullFee(args)
+  }, [args, getFullFee])
+
+  return {
+    fullFee,
+    getBasicFee,
+    hasEnoughFunds: fullFee > (accountBalance || 0),
+    loading,
+  }
+}
+
+export const useBloatFeesAndPerMbFees = (assets?: VideoInputAssets | ChannelInputAssets) => {
+  const {
+    chainState: {
+      dataObjectPerMegabyteFee,
+      dataObjectStateBloatBondValue,
+      channelStateBloatBondValue,
+      videoStateBloatBondValue,
+    },
+  } = useJoystream()
+
+  const getFeePerMb = useCallback(
```

Comment:
This name and the property the value is assigned to `feePerMb` is misleading. This  value isn't fee per MB, it's total fee related to assets size, already taking the size into account so no "per MB" anymore